code/game/gamemodes/clock_cult/clock_structures/tinkerers_cache.dm

```diff
@@ -18,7 +18,9 @@
 	START_PROCESSING(SSobj, src)
 	clockwork_caches++
 	update_slab_info()
-	SetLuminosity(2,1)
+	#warn falloff?
```

Comment:
I was midway through this and I didn't want to ask on IRC what the second argument was to ruin the surprise so.